### DISCUSSION TITLE
Mac端Telegram无法正常连接，Win端可以正常使用。

### DISCUSSION THREAD

**Discussion by Obi-Z7** [clash-verge-rev/clash-verge-rev#2132] [Q&A]
请问我在使用Mac端系统代理的时候，同样的节点以及设置，无法连接Telegram，但是Win端却可以正常连接没有任何问题，这个是什么原因呢？

**Comment by Dragon1573** [comment]
Telegram 的代理设置需要在其内部另外调。是不是你 macOS 上没有让 Telegram 使用系统代理？

<div align="center"><p>
<img width="473" height="703" alt="PixPin_2025-07-13_22-52-27" src="https://github.com/user-attachments/assets/9277c6a0-3d4a-4091-a64d-f2d89b246580" />
</p></div>
